docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -4,19 +4,26 @@ title: Build a logs data source plugin
 
 # Build a logs data source plugin
 
-This guide explains how to build a logs data source plugin.
+This guide explains how to build a logs data source plugin. 
 
-Data sources in Grafana supports both metrics and log data. The steps to build a logs data source plugin are largely the same as for a metrics data source. This guide assumes that you're already familiar with how to [Build a data source plugin](/tutorials/build-a-data-source-plugin/) for metrics.
+Data logs are defined as data stored over a period of time in different systems or environments for the purpose of tracking a variety of events.  
+
+Grafana data source plugins support both metrics and log data. The steps to build a logs data source plugin are largely the same as for a metrics data source, but there are a few differences. 
+
+## Before you begin
+
+This guide assumes that you're already familiar with how to [Build a data source plugin](/tutorials/build-a-data-source-plugin/) for metrics. We recommend that you review this material before continuing.
 
 ## Add logs support to your data source
 
 To add logs support to an existing data source, you need to:
 
-- Enable logs support
-- Construct the log data
-- (Optional) Add preferred visualisation type hint to the data frame
+1. Enable logs support
+1. Construct the log data
 
-### Enable logs support
+When these steps are done, then you can improve the user experience with one or more optional features.
```

Comment:
Should you link to the optional features?

Reply:
Done, will check in soon